### PR TITLE
 Interoperability: Changes to Data Privacy (EXPOSUREAPP-2954)

### PR DESCRIPTION
**Pull request by harambasicluka**
Updated german data privacy and copied it over to turkish and english until we have the final ones.

### CODE REVIEW

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 1]
```diff
@@ -100,479 +122,465 @@ <h2>
...
-    <li>
-        Andere warnen (Übermitteln Ihrer Zufalls-IDs)
-    </li>
-</ul>
-<p>
-    Die Zugriffsdaten umfassen folgende Daten:
+    Bei jedem Internet-Datenaustausch der App mit dem Serversystem des RKI (im
+    Folgenden: <strong>Serversystem</strong>) werden vom Serversystem
+    sogenannte Zugriffsdaten verarbeitet. Dies ist erforderlich, damit die App
+    aktuelle Daten (z. B. für Warnungen) abrufen oder bestimmte auf dem
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/privacy_de.html:128]:
```suggestion
    aktuelle Daten (z.B. für Warnungen) abrufen oder bestimmte auf dem
```

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 2]
```diff
@@ -100,479 +122,465 @@ <h2>
...
-    einer bestimmten Person enthalten. Dazu gehören nicht nur Angaben zu
-    früheren und aktuellen Krankheiten, sondern auch zu Krankheitsrisiken einer
-    Person (z. B. das Risiko, dass eine Person sich mit dem Corona-Virus
-    infiziert hat).
-</p>
-<p>
-    Bei der Nutzung folgender Funktionen verarbeitet die App Ihre
-    Gesundheitsdaten:
+    einer Person enthalten. Dazu gehören nicht nur Angaben zu früheren und
+    aktuellen Krankheiten, sondern auch zu Krankheitsrisiken einer Person (z.
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/privacy_de.html:235]:
```suggestion
    aktuellen Krankheiten, sondern auch zu Krankheitsrisiken einer Person (z.```

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 3]
```diff
@@ -100,479 +122,465 @@ <h2>
...
-    resultierender fehlerhafter Warnungen und Risikowerte. Nach ausreichender
-    Beantwortung dieser Fragen werden Sie nach Ihrer Handy-/Telefonnummer
-    gefragt. Dies dient dazu, Sie später zurückrufen zu können, um Ihnen eine
-    TeleTAN zur Eingabe in der App mitzuteilen. Ihre Handy-/Telefonnummer wird
-    nur zu diesem Zweck vorübergehend gespeichert und spätestens innerhalb
-    einer Stunde gelöscht.
+    Soweit Sie die App nur informatorisch nutzen, also keine der oben genannten
+    Funktionen verwenden, findet die Verarbeitung ausschließlich lokal auf
+    Ihrem Smartphone statt und es werden keine personenbezogenen Daten durch
+    das RKI verarbeitet. In der App verlinkte Webseiten, z. B.: www.bundesregierung.de, werden im
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/privacy_de.html:461]:
```suggestion
    das RKI verarbeitet. In der App verlinkte Webseiten, z.B.: www.bundesregierung.de, werden im
```

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 4]
```diff
@@ -582,207 +590,153 @@ <h2>
...
-    </li>
-    <li>
-        Das Token, das in der App gespeichert ist, wird nach Löschung der App
-        vom Smartphone oder nach Ausführung der Funktion „Andere warnen“
-        gelöscht.
-    </li>
-</ul>
+<p>
+    Die abgerufenen Positiv-Listen werden nach 14 Tagen aus dem App-Speicher
+    gelöscht. Das in der App ermittelte Infektionsrisiko (z. B. „niedriges
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/privacy_de.html:627]:
```suggestion
    gelöscht. Das in der App ermittelte Infektionsrisiko (z.B. „niedriges
```

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 5]
```diff
@@ -582,207 +590,153 @@ <h2>
...
+    Positiv-Listen an ihre jeweiligen eigenen Nutzer.
+</p>
+<p>
+    Mit dem Betrieb und der Wartung des gemeinsam betriebenen Warnsystems haben
+    die zuständigen nationalen Gesundheitsbehörden der teilnehmenden Länder die
+    EU-Kommission als Auftragsverarbeiter beauftragt.
 </p>
 <p>
     Mit dem Betrieb und der Wartung eines Teils der technischen Infrastruktur
     der App (z. B. Serversysteme, Hotline) hat das RKI die T-Systems
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/privacy_de.html:660]:
```suggestion
    der App (z.B. Serversysteme, Hotline) hat das RKI die T-Systems
```

**Review by ralfgehrer — APPROVED** [review]

**Comment by harambasicluka** [comment]
Will merge for now, if we agree on changing this we'll do it next week.

**Pull request merged**